saviSimulation/savi/jason_processing/behaviour/UaVBehavior.java

```diff
@@ -0,0 +1,43 @@
+package savi.jason_processing.behaviour;
+
+import processing.core.*;
+
+
+public class UaVBehavior extends UxVBehavior {
+    private static final float VERTICAL_SPEED = 0.1f; // 0.1 something (whatever real-life distance this corresponds to)
```

Comment:
so just a thought here: Should UaV perhaps have a different (faster) speed than UgV? 
I know this isn't an issue introduced in this PR but since it comes to mind I may as well bring it up. Also, off hand I'm unsure how polymorphism works with static variables, so if we do introduce a different speed for different subclasses of UxV I'm unsure how that affects the AgentAction created within UxV (so I'd like a confirmation that this won't be an issue).

Reply:
Aside from this potential issue I'm happy with the changes.
It's not the most legible code (for people less experienced with Java) but perhaps a couple of lines of comment above the relevant code could help explain what's going on.

Reply:
Yep, I agree with you.

The speed would be a good thing to introduce as a field of the UxVBehaviour class rather than as a constant. This would allow each implementation of UxVBehaviour to have their speeds vary. 

It shouldn't create any issues with the AgentAction since we could access the speed values as 'this.speedVal'.

Reply:
👍

Reply:
Made some changes to accommodate this. 

Take a look at the constructors of UaVBehaviour and UgVBehaviour and how they initialize their own 'maxSpeed' using the super-class constructor. Let me know your thoughts!